content/en/docs/plugins/plugin-api-reference.md

```diff
@@ -84,6 +88,8 @@ When managing plugin-level state, keep the following in mind:
 
 The format of the config string is entirely determined by the plugin author, and by default is passed unchanged from Falco/the application using the plugin framework to the plugin. However, semi-structured formats like JSON/YAML are preferable to free-form text. In those cases, the plugin author can provide a schema describing the config string contents by implementing the optional `get_init_schema` function. If so, the `init` function can assume the passed-in configuration string to always be well-formed, and can avoid performing any error handling. The plugin framework will take care of automatically parsing it against the provided schema and generating ad-hoc errors accordingly. Please refer to the documentation of `get_init_schema` for more details.
 
+If a non-NULL ss_plugin_t* state is returned, then subsequent invocations of `init` must not return the same `ss_plugin_t *` value again, if not after it has been disposed with `destroy` first.
```

Comment:
Not sure I understand the meaning of *if not after it has been disposed with `destroy` first.*
Do you maybe mean 'unless'?